mbed_flasher/erase.py

```diff
@@ -234,12 +235,12 @@ def _erase_board_simple(self, target, no_reset):
         while auto_thread.is_alive():
             auto_thread.join(0.5)
 
-        new_target = MountVerifier(self.logger).check_points_unchanged(target)
-        if isinstance(new_target, int):
-            return new_target
+        target = FlasherMbed.refresh_target(target["target_id"])
```

Comment:
Is that removing MountVerifier becuase mbed-ls has done such checks?
Does it mean we have to depend more on mbed-ls?

Reply:
Yes, flasher should not need to do those checks.